assets/_sass/custom-viz/boeing/_scrollytelling.scss

```diff
@@ -19,7 +19,7 @@
   .svgLabel {
     visibility: hidden;
     opacity: 0;
-    transition: visibility 0.7s linear, opacity 0.7s linear;
+    transition: visibility 0.4s linear, opacity 0.7s linear;
```

Comment:
Don't forget to change the `opacity` duration value too!

Reply:
whoops!